content/03.use-cases-a.md

```diff
@@ -34,16 +25,16 @@ Some EEB labs have even turned their lab notebooks into shareable websites [@url
-While there are other purpose-built platforms for this, GitHub provides a free, open-source alernative.
+While there are other purpose-built platforms for this, GitHub provides a free, open-source alternative.
 
 GitHub's open nature allows colleagues to see, review and offer feedback on your process.
-Making presentations can be done through most major high-level programming languages such as R, with `RMarkdown` [@isbn:9781138359338], Python, with `python-ppt` [@https://python-pptx.readthedocs.io/en/latest/index.html]), and Julia, with `Remark.jl` [@https://juliapackages.com/p/remark].
+Making presentations can be done through most major high-level programming languages such as R, with `RMarkdown` [@isbn:9781138359338]; Python, with `python-ppt` [@https://python-pptx.readthedocs.io/en/latest/index.html]); and Julia, with `Remark.jl` [@https://juliapackages.com/p/remark].
 Since all these programs work via code bases, they can be version-controlled through git and GitHub.
-Once content is made, hosting a course website can be done through GitHub Pages, and there are lots of templates available (e.g., see [@https://github.com/topics/course-website]).
+Once content is made, hosting a course website can be done through GitHub Pages, and there are numerous templates available (e.g., see [@https://github.com/topics/course-website]).
 This way, not only can the course content be available to enrolled students, but also to a global pool of learners and teachers interested in the course material.
 Content can then be delivered via the course website, and/or a GitHub organization with, for example, template repositories for assignments.
 Student submissions are perhaps the most challenging component, but for assignments submitted as code files (i.e., `.R` & `.Rmd` as two of the most common) and/or `.pdf` files, GitHub has a new tool in development  - GitHub classroom [@https://classroom.github.com] - where instructors can host private assignments, and even build custom autograding tests.
  
 
 While most instructors can feasibly only incorporate some of these tools to deliver their content to students, it is still valuable to do so and to encourage students to begin learning about version control through interacting with git/GitHub, however minimally, through the course.  
-Especially if a central tenet of a given course or educational unit is to introduce or give students experience to version control and the tools that working professionals in EEB use, then adopting a few of these tools can be a great way to do so.
+Especially if a central tenet of a given course or educational unit is to introduce or give students experience with version control and the tools that working professionals in EEB use, then adopting a few of these tools can be a great way to do so.
```

Comment:
>    Is this the main benefit over, e.g., Canvas? If so, this needs to be in the first paragraph of the section.

Reply:
Agree, I think we should lead with at least the potential benefits/reasons why people might switch/complement their work in canvas with a course on GitHub. Essentially, lead with the 'why'